tests/Ethereum/AbiTests.cpp

```diff
@@ -962,38 +995,35 @@ TEST(EthereumAbi, DecodeParamsMixed) {
 
 TEST(EthereumAbi, EncodeSignature) {
     auto func = Function("baz", std::vector<std::shared_ptr<ParamBase>>{
-        std::make_shared<ParamUInt256>(69u),
-        std::make_shared<ParamBool>(true)
-    });
+                                    std::make_shared<ParamUInt256>(69u),
+                                    std::make_shared<ParamBool>(true)});
     EXPECT_EQ("baz(uint256,bool)", func.getType());
     Data encoded;
     func.encode(encoded);
 
     EXPECT_EQ(encoded.size(), 32 * 2 + 4ul);
     EXPECT_EQ(hex(encoded.begin(), encoded.begin() + 4), "72ed38b6");
-    EXPECT_EQ(hex(encoded.begin() +   4, encoded.begin() + 36 ), "0000000000000000000000000000000000000000000000000000000000000045");
-    EXPECT_EQ(hex(encoded.begin() +  36, encoded.begin() + 68 ), "0000000000000000000000000000000000000000000000000000000000000001");
+    EXPECT_EQ(hex(encoded.begin() + 4, encoded.begin() + 36), "0000000000000000000000000000000000000000000000000000000000000045");
+    EXPECT_EQ(hex(encoded.begin() + 36, encoded.begin() + 68), "0000000000000000000000000000000000000000000000000000000000000001");
 }
 
 TEST(EthereumAbi, EncodeFunctionWithDynamicArgumentsCase1) {
     auto func = Function("sam", std::vector<std::shared_ptr<ParamBase>>{
-        std::make_shared<ParamByteArray>(Data{0x64, 0x61, 0x76, 0x65}),
-        std::make_shared<ParamBool>(true),
-        std::make_shared<ParamArray>(std::vector<std::shared_ptr<ParamBase>>{
-            std::make_shared<ParamUInt256>(1),
-            std::make_shared<ParamUInt256>(2),
-            std::make_shared<ParamUInt256>(3)
-        })
-    });
```

Comment:
These indentations are nicer, can we tweak clang-format?

Reply:
Yes I can turn it off for a scope, from the config it maybe a bit complex